### PR TITLE
A Suggestion to improve the load of the login.py

### PR DESCRIPTION
**Pull request by spbonez**

### CODE REVIEW

File: config/cmd_not_found.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cmd_not_found_messages = ["I'm sorry Dave, I'm afraid I can't do that", "It can only be attributable to human error.",
```
Comment (spbonez) [inline config/cmd_not_found.py]: Adding this to it's own file clean up the code of login.py

File: src/login.py [hunk 2]
```diff
@@ -12,24 +13,7 @@
 
 def command_not_found():
 
-    messages = ["I'm sorry Dave, I'm afraid I can't do that", "It can only be attributable to human error.",
```
Comment (spbonez) [inline src/login.py]: removing the messages here and placing them in a new file in config will clean up the code.

File: src/login.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 import os
 from discord.ext import commands
 from config.auth import User
+from config.cmd_not_found import cmd_not_found_messages
```
Comment (spbonez) [inline src/login.py]: calling the messages from config to here

**Comment by Luke0129** [comment]
Good suggestion. but currently, the messages don't even get used. And when they do, there is 70% is not going to be in login.py.

**Pull request merged**